app.js

```diff
@@ -76,8 +103,51 @@ app.get("*", function (req, res) {
   res.render("notFound");
 });
 
+//Register route
+app.get('/register', (req, res) => {
+  
+});
+
+app.post('/register', async (req, res) => {
+
+  try {
+    const {
+      password,
+      email,
+      name,
+      organization,
+      register_ip,
+      create_date
+    } = req.body.user;
+    //hashing the password
+    const hash = await bcrypt.hash(password, 12);
+   
+   
+    // IP Middleware
+  //   app.use(function(req, res) {
+  //     //const ip = req.clientIp;
+  //     res.end(ip);
+  // });
+
+   
+    // date
+    const createDate = moment().format('MM/DD/YYYY');
+    
+    //creating a user
+    const results = await db.query('INSERT INTO users(email, password, name, organization, register_ip, create_date) VALUES($1, $2, $3, $4, $5, $6)', [email, hash, name, organization, req.clientIp, createDate]);
+    
+    res.status(201).send(`User added with ID: ${results.insertId}`);
```

Comment:
Nice use for 201!

Reply:
However, does it work results.insertId? I think you need results.rows[0].id? or something else?